modules/tags.js

```diff
@@ -0,0 +1,235 @@
+  if (!content && !attachment) return int.editReply("I either need content or a file.");
+  // Create the tag
+  const command = await u.db.tags.manageTag({
+    tag: name,
+    response: content ?? undefined,
+    attachment: attachment?.name
+  });
+  if (!command) return int.editReply("I wasn't able to save that. Please try again later or with a different name.");
+  if (attachment) saveAttachment(attachment, command);
+
+  tags.set(command.tag, command);
+  const embed = u.embed({ author: int.member })
+    .setTitle("Tag created")
+    .setDescription(`${int.member} added the tag \`${name}\``);
+  if (command.response) embed.addFields({ name: "Response", value: command.response });
+  try {
+    // report the tag creation
+    int.client.getTextChannel(u.sf.channels.team)?.send({ embeds: [embed], files: attachment ? [attachment] : [] });
+    int.editReply({ content: "Tag created!", embeds: [embed.setDescription(`Try it out with \`!${name}\``)], files: attachment ? [attachment] : [] });
+  } catch (error) {
+    int.client.getTextChannel(u.sf.channels.team)?.send({ embeds: [embed.setFields({ name: "Error", value: "The tag creation preview was too long to send." })] });
+    int.editReply(`I saved the tag \`${name}\`, but I wasn't able to send you the preview`);
+  }
+}
+
+/** @param {Augur.GuildInteraction<"CommandSlash">} int */
+async function slashTagModify(int) {
+  // get and validate inputs
+  const name = int.options.getString('name', true).toLowerCase().replace("\n", "");
+  const content = int.options.getString('content');
+  const attachment = int.options.getAttachment('attachment');
+  const currentTag = findTag(name);
+  if (!currentTag) return int.editReply(`Looks like that tag doesn't exist.`);
+  if (!content && !attachment) return int.editReply(`I need a response, a file, or both. If you want to delete the tag, use </tag delete:<${u.sf.commands.slashTag}>.`);
+
+  // modify the tag
+  const command = await u.db.tags.manageTag({
+    tag: name,
+    response: content ?? undefined,
+    attachment: attachment?.name
+  });
+
+  if (!command) return int.editReply("I wasn't able to update that. Please try again later or contact a dev to see what went wrong.");
+  if (attachment) saveAttachment(attachment, command);
+
+  tags.set(command.tag, command);
+  const embed = u.embed({ author: int.member })
+    .setTitle("Tag modified")
+    .setDescription(`${int.member} modified the tag "${name}`);
+
+  // log the modification
+  try {
+    if (command.response != currentTag.response) {
+      embed.addFields(
+        { name: "Old Response", value: currentTag.response ?? 'None' },
+        { name: "New Response", value: command.response ?? 'None' },
+        { name: "Attachment Status", value: currentTag.attachment ? attachment ? "Removed" : "Replaced" : attachment ? "Added" : "Unchanged" }
```

Comment:
Not sure if 137 is working the way that you were wanting. 
![image](https://github.com/LDS-Gamers-Studios/icarus5.5/assets/79873150/a8992561-adb1-4ff7-a51e-f1c0858e32b6)
This changed the text but not the image.
I'm not sure if there's some way to remove the attachment but I'm not sure.